collect_app/src/main/java/org/odk/collect/android/activities/GeoPointMapActivity.java

```diff
@@ -102,12 +102,12 @@
     /** While true, the point cannot be moved by dragging or long-pressing. */
     private boolean isPointLocked;
 
-    private Bundle instanceState;
+    private Bundle previousState;
```

Comment:
Can't we move this field to `BaseGeoMapActivity`, make protected and use in other classes instead?

Reply:
Sure!

Reply:
Done.